evap/evaluation/templates/email_base.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html style="background-color: #495057;">
+
+<head>
+  <meta charset="utf-8"/>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+  <meta name="viewport" content="width=device-width; initial-scale=1; shrink-to-fit=no"/>
+  <title>EvaP</title>
+</head>
+
+<body style="margin: 0px; padding: 0px; font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'; font-size: 1rem; font-weight: 400; line-height: 1.5; color: #212529; background-color: #ffffff;">
+  <table style="background-color: #495057; width: 100%; border-collapse: collapse;" cellspacing="0" cellpadding="0">
+    <tr>
+      <td>
+        <a href="{{ page_url }}">
+          <img src="https://evap.hpi.de/static/images/evap.png" height="50" style="display: block;"/>
+        </a>
+      </td>
```

Comment:
Do we need this empty td? I don't see how it could change anything? Maybe we could remove the whole table if just one row and one column is left?

Reply:
You are right, it is not needed. But replacing the table with e.g. a `div` sadly breaks the template on my mobile email client, so I'm going to leave the table in.